src/panels/config/hardware/ha-config-hardware.ts

```diff
@@ -50,9 +71,87 @@ class HaConfigHardware extends LitElement {
 
   @state() private _hardwareInfo?: HardwareInfo;
 
+  @state() private _chartOptions?: ChartOptions;
+
+  @state() private _systemStatusData?: SystemStatusStreamMessage;
+
+  private _memoryEntries: { x: string; y: number | null }[] = [];
+
+  private _cpuEntries: { x: string; y: number | null }[] = [];
+
+  private _subscribed?: Promise<(() => Promise<void>) | void>;
+
+  private _unsubscribe(): void {
+    if (this._subscribed) {
+      this._subscribed.then((unsub) =>
+        unsub ? unsub().catch(() => {}) : undefined
+      );
+      this._subscribed = undefined;
+    }
+  }
+
+  public connectedCallback() {
+    super.connectedCallback();
+    if (this.hasUpdated) {
+      this._subscribeSystemStatus();
+    }
+  }
+
+  public disconnectedCallback() {
+    super.disconnectedCallback();
+    this._memoryEntries = [];
+    this._cpuEntries = [];
+    this._unsubscribe();
+  }
+
+  protected willUpdate(): void {
+    if (!this.hasUpdated) {
+      this._chartOptions = {
```

Comment:
this is not dynamic? So it can just be a const outside the class?

Reply:
Now its dynamic :)